pkgs/development/python-modules/cle/default.nix

```diff
@@ -38,7 +38,7 @@ buildPythonPackage rec {
     owner = "angr";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-qJrVAofIhoYS9dA/ipLh6Xuyt2b+trDRE/yq8tV9arI=";
+    hash = "sha256-6NDNGcKiBKKzkzXx/gnSyO/dCHD4VrcClGCMndwxF9Q=";
```

Comment:
The hash above (for binaries) should also be updated

Reply:
You are right. Thanks for checking.